sql/upgrade/sl3.0.sql

```diff
@@ -903,60 +910,71 @@ ALTER TABLE gl ENABLE TRIGGER gl_audit_trail;
 
 ALTER TABLE ar DISABLE TRIGGER ar_audit_trail;
 
---TODO: Handle amount_tc and netamount_tc
 insert into ar
-(entity_credit_account, person_id,
+        (entity_credit_account, person_id,
         id, invnumber, transdate, taxincluded,
         amount_bc, netamount_bc,
         amount_tc, netamount_tc,
 <?lsmb IF VERSION_COMPARE(lsmbversion,'1.6') < 0; ?>
         paid, datepaid,
 <?lsmb END; ?>
```

Comment:
The following SELECT doesn't have a conditional block like this. Do you expect this PR to be backported to 1.6? If not, wouldn't it be better not to have any conditionals on the target schema? (It's complex enough as it is when you want to support multiple origin schemas...)